variables.tf

```diff
@@ -139,3 +139,21 @@ variable "labels" {
   description = "Key-value pairs representing labels attaching to instance & instance template"
   default     = {}
 }
+
+variable "google_logging_enabled" {
+  type        = bool
+  description = "Enable Google Cloud Logging"
```

Comment:
What does it exactly enable? As logging already works, if I look a Cloud Logging I can already see container logs being written there.

Reply:
Logging is being changed and these settings should allow to make some choices that will enforce settings even when defaults will change and/or to disable some features